pandas/core/format.py

```diff
@@ -1640,11 +1644,14 @@ class ExcelFormatter(object):
     inf_rep : string, default `'inf'`
         representation for np.inf values (which aren't representable in Excel)
         A `'-'` sign will be added in front of -inf.
+    verbose: boolean, default True
+             If True, warn user that the resulting output file may not be
+             re-read or parsed directly by pandas.
```

Comment:
can you use normal indentation (4 spaces here)? no alignment with the 'boolean, ..' is needed. (se the other parameters)